app/assets/stylesheets/mail/email.scss

```diff
@@ -121,6 +121,10 @@ table.order-summary {
   text-align: right;
 }
 
+.middle {
+  vertical-align: middle;
+}
+
```

Comment:
I don't see this CSS class used anywhere. Did you commit this accidentally?

Reply:
Thanks, Maikel! You're right, the application of this class got lost when I rebased onto Ahmed's PR to fix images in emails. I've re-applied the class and found it could be even more simplified, as you will see in my new PR. Unfortunately, I accidentally deleted the branch and this original PR here got closed when I re-installed my system, therefore I am opening a new one.